lib/planner/write-plans-to-disk.js

```diff
@@ -4,14 +4,14 @@ const fs = require('fs');
 const path = require('path');
 const configHandler = require('../util/config/config-handler.js');
 
+
 module.exports = function writePlansToDisk(plans) {
-  let date = Date.now();
-  let fileCount = 1;
   const outputPath = configHandler.get('outputPath');
   for (let plan of plans) {
-    let filePath = path.resolve(outputPath,
-        `${date}-simulato-${fileCount++}_${plans.length}.json`);
-    // fs.writeFileSync(filePath, JSON.stringify(plan.testCase));
+    const crypto = require('crypto');
```

Comment:
Most all `require`s should be at the top of the file unless there is a reason not to.